packages/suite/src/components/wallet/AccountsMenu/index.tsx

```diff
@@ -134,7 +135,11 @@ const AccountsMenu = ({ device, accounts, selectedAccount }: Props) => {
     const theme = useTheme();
     const { discovery, getDiscoveryStatus } = useDiscovery();
     const { params } = selectedAccount;
-    const { isMobileLayout } = useLayoutSize();
+    const { isMobileLayout, layoutSize } = useLayoutSize();
+    // Must be kept in sync with SuiteLayout.
+    const isInline = useSelector(
+        state => isMobileLayout || (layoutSize !== 'XLARGE' && state.guide.open),
+    );
```

Comment:
I don't like this. But I'm not sure how to do this better.

It would be IMO best to pass the `isInline` as a prop. However I didn't find out how that can be done in `SuiteLayout` where TS doesn't know that `menu` is actually `AccountsMenu`. I'm open to any suggestion on how to do this nicer.

Reply:
I think AccountsMenu should have an access to `LayoutContext.Provider`? So you can define `isInline` in the `SuiteLayout` file and pass it here. 

(I am not 100% sure, it is hard to get into this code without spending a lot of time on it.)

Reply:
@tomasklim Thanks a lot! I had to educate myself on what React Context is but now I think I cracked it. Please check it out.
03251f41d6cef02e44dd1a22ac4a53ac34ae85e9